docs/UserGuide.md

```diff
@@ -136,6 +136,36 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd patient in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st patient in the results of the `find` command.
 
+### Adding an appointment: `assign`
+
+Assign the specified patient into the specified appointment date and time.
+
+Format: `assign INDEX d/DATE t/TIME`
+
+* Puts the person at the specified INDEX into an appointment time slot.
```

Comment:
Small nits but will it be better to use the word patient here instead?

Reply:
Oh yeah. I missed that out. Thanks 👍